components/common/mou-list.json

```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_common_mou_files",
+  "info": {
+    "name": "mou_file",
+    "icon": "images",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "mou_file": {
+      "model": "file",
+      "via": "related",
+      "allowedTypes": [
+        "images",
+        "files",
+        "videos"
+      ],
+      "plugin": "upload",
+      "required": false,
+      "pluginOptions": {}
+    },
+    "start_date": {
+      "type": "date"
```

Comment:
added type: date as previously, it was dateTime and was giving the upload time as well.

Reply:
is there a problem with upload time also? I would actually prefer that if we have time as well

Reply:
there is no problem with that, I just thought that we might only need the date for uploading.